includes/subscriptions/class-wc-payments-subscription-service.php

```diff
@@ -1004,4 +1004,26 @@ private function validate_subscription_data( $subscription_data ) {
 			}
 		}
 	}
+
+	/**
+	 * Determines if the store has any active WCPay subscriptions.
+	 *
+	 * @return bool True if store has active WCPay subscriptions, otherwise false.
+	 */
+	public static function store_has_active_wcpay_subscriptions() {
+		$results = wcs_get_subscriptions(
+			[
+				'subscriptions_per_page' => 1,
+				'fields'                 => 'ids',
```

Comment:
`'fields'` isn't a valid param for `wcs_get_subscriptions()` so this can be removed. This function always returns the object.

Reply:
I saw that eventually `wc_get_orders` was called, so assumed the args would carry through. Fixed in 14ef0a4974d1e21ff79202c30fd58bd03f7bff07